metadata_tools/geometry_support.py

```
@@ -225,10 +430,29 @@ def _meshgrid(self, observation, meshgrids):
+        """
+        if body in oops.Body.BODY_REGISTRY:
+            parent = oops.Body.BODY_REGISTRY[body].parent.name
+        else:
+            return []
+        if parent != 'SUN':
+            return parent
+        return body
+
     #===============================================================================
     def add(self, qualifier, *,
                   name=None, target=None, tiles=[], tiling_min=100,
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)